ricecooker/classes/files.py

```diff
@@ -735,6 +751,8 @@ def process_file(self):
 
         return self.filename
 
+    def get_duration(self):
```

Comment:
I think to simplify things, we can just remove these `get_duration` methods from the file classes - they are never referred to anywhere, and just setting the `duration` property should be sufficient.